### ISSUE TITLE
Can I use your lib like so?

### ISSUE THREAD

**Issue by venomone** [mueslimak3r/tv-intro-detection#34]
This is not really a BUG report, it's more a question. I'm working on a VoD white label product. The last feature my player is missing, is the Skip Intro feature. Now I came across your nice script and I have to say that it really does not fit my implementation as it's all basically embedded into a Django Rest Application.

My Idea is to first save the hash for each of my videos during my normal import workflow I already have. Afterward, I have the hash of each of my clips inside my Database. Now if the user plays episode 3 for example, I would simply compare the hash from episode 3 with the hash of episode 4 or 2. Is this theoretically working with your code?

Can you tell me, in what function you create the hash and in which you compare it with the other, basically this is everything I need (I guess).

Many thanks in advance, and also thanks for your work, really appreciate that.

**Comment by mueslimak3r** [comment]
Yes, the script is designed to work as you described. Hashes can be saved and used later for comparisons instead of processing all videos each time.

I think the logic for the "season hash" is what you're looking for.

Please keep in mind that the license of this project requires the public disclosure of source code that incorporates the project, and requires that the same license or a compatible license is used.

I recommend that you go over the all terms of the license before incorporating it into a commercial product.

I'm happy to provide more details if needed. 🙂
